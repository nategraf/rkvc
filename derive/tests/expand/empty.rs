use rkvc_derive::Attributes;

#[derive(Attributes)]
struct EmptyStruct {}

fn main() {}
