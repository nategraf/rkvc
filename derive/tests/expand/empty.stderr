error: proc-macro derive panicked
 --> tests/expand/empty.rs:3:10
  |
3 | #[derive(Attributes)]
  |          ^^^^^^^^^^
  |
  = help: message: not implemented: Not implemented for empty structs
